cargo/lib/dependabot/cargo/update_checker/latest_version_finder.rb

```diff
@@ -69,26 +69,43 @@ def filter_lower_versions(versions_array)
         end
 
         def available_versions
-          crates_listing.
-            fetch("versions", []).
-            reject { |v| v["yanked"] }.
-            map { |v| version_class.new(v.fetch("num")) }
+          # Sparse registries return listings in different formats.
```

Comment:
Technically this isn't a difference between sparse indexes and bare, those have the same format, this is a difference between indexes and the undocumented `/api/v1/crate/<name>` api.

Reply:
Ah, I see! I'll adjust the comment